pkg/agent/endpoints/workload/handler_test.go

```diff
@@ -665,13 +685,11 @@ func TestFetchJWTSVID(t *testing.T) {
 						assert.Nil(t, resp)
 						return
 					}
-					var tokenIDs []spiffeid.ID
-					for _, svid := range resp.Svids {
-						parsedSVID, err := jwtsvid.ParseInsecure(svid.Svid, tt.audience)
-						require.NoError(t, err, "JWT-SVID token is malformed")
```

Comment:
I think we still want these verifications to make sure the JWT itself is well-formed

Reply:
https://github.com/spiffe/spire/commit/435498e74f5a8c107b12168c25e651557f46e405